src/features/game/types/removeables.ts

```diff
@@ -184,6 +184,15 @@ function areAnyTreasureHolesDug(game: GameState): Restriction {
   return [holesDug, "Treasure holes are dug"];
 }
 
+function areAnyComposting(game: GameState): Restriction {
+  return [
+    isComposting(game, "Compost Bin") ||
+      isComposting(game, "Turbo Composter") ||
+      isComposting(game, "Premium Composter"),
+    "In use",
```

Comment:
Might be good to change the text to "Composters in use" to Standardise with other removal restriction texts